brigade.js

```diff
@@ -72,6 +72,32 @@ const lintJSJob = (e, p) => {
 }
 jobs[lintJSJobName] = lintJSJob;
 
+const testIntegrationJobName = "test-integration";
+const testIntegrationJob = (e, p) => {
+  let kind = new KindJob(testIntegrationJobName);
+  kind.mountPath = localPath;
+  kind.tasks.push(
+    // Install git and golang deps
+    "apk add --update --no-cache git gcc musl-dev",
+    // Install helm
+    `curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 \
+      && chmod 700 get_helm.sh \
+      && ./get_helm.sh`,
```

Comment:
Non-blocking, but do we want to consider bundling all relevant tools into an image?

Reply:
Yea, what do we think?  PR to bake them into the image that brigade-utils creates for the default KindJob image? Or create a new one in brigadecore that builds on that default image?  I think I might prefer the former, since this job (and others in brigade-utils) are really pretty opinionated for our uses anyhow....

Reply:
No strong opinion on which way to go.

I do have an opinion about us having opinions tho...

I think that with both 1.x and now 2.x, we're uncovering a lot of useful patterns that others may seek to emulate, and may be worth documenting (or documenting further) and if we were to curate some useful images that facilitate those patterns, people would probably use them.